.github/workflows/download-component.yml

```
@@ -0,0 +1,70 @@
+      component_version:
+        description: "The resolved component version"
+        value: ${{ jobs.download.outputs.component_version }}
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+    outputs:
+      component_version: ${{ steps.resolve_version.outputs.version }}
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)